wemake_python_styleguide/cli/commands/explain/command.py

```diff
@@ -0,0 +1,44 @@
+"""Contains command implementation."""
+
+from wemake_python_styleguide.cli.commands.base import AbstractCommand
+from wemake_python_styleguide.cli.commands.explain import (
+    message_formatter,
+    violation_loader,
+)
+from wemake_python_styleguide.cli.output import print_stderr, print_stdout
+
+
+def _clean_violation_code(violation_str: str) -> int:
+    """
+    Get int violation code from str violation code.
+
+    Args:
+        violation_str: violation code expressed as string
+                       WPS412, 412 - both acceptable
+
+    Returns:
+        integer violation code
+
+    Throws:
```

Comment:
No, it does not throw. I propose to remove this long docstring for such a short function.

Reply:
Just keep `"Get int violation code from str violation code."` part